assets/utils/string.js

```diff
@@ -1,2 +1,7 @@
 export const camelToSnakeCase = (str) => str.replace(/[A-Z]/g, (letter) => `_${letter.toLowerCase()}`);
+
+export const snakeToCamel = (str) => str.toLowerCase()
```

Comment:
pour garder la cohérence avec `camelToSnakeCase` je propose garder le suffixe Case pour cette fonction aussi :

```suggestion
export const snakeToCamelCase = (str) => str.toLowerCase()
```